src/hooks/useAllLiquidityMiningCampaigns.ts

```diff
@@ -261,52 +256,12 @@ export function useAllLiquidtyMiningCampaigns(
       }
     }
 
-    const sortedActiveCampaigns = activeCampaigns.sort((a, b) => {
-      if (a.campaign.ended && !b.campaign.ended) return -1
-      if (!a.campaign.ended && b.campaign.ended) return 1
-
-      if (a.staked && !b.staked) return -1
-      if (!a.staked && b.staked) return 1
-
-      if (
-        a.campaign instanceof SingleSidedLiquidityMiningCampaign &&
-        !(b.campaign instanceof SingleSidedLiquidityMiningCampaign)
-      )
-        return -1
-
-      if (
-        !(a.campaign instanceof SingleSidedLiquidityMiningCampaign) &&
-        b.campaign instanceof SingleSidedLiquidityMiningCampaign
-      )
-        return 1
-
-      // Active above upcoming
-      if (a.campaign.currentlyActive && !b.campaign.currentlyActive) return -1
-      if (!a.campaign.currentlyActive && b.campaign.currentlyActive) return 1
-
-      // TV
-      if (a.campaign.staked.nativeCurrencyAmount.greaterThan(b.campaign.staked.nativeCurrencyAmount)) return -1
-      if (a.campaign.staked.nativeCurrencyAmount.lessThan(b.campaign.staked.nativeCurrencyAmount)) return 1
-
-      if (a.campaign.apy > b.campaign.apy) return -1
-      if (a.campaign.apy < b.campaign.apy) return 1
-
-      return 0
-    })
-
-    const sortedExpiredCampaigns = expiredCampaigns.sort((a, b) => {
-      if (a.campaign.endsAt > b.campaign.endsAt) return -1
-      if (a.campaign.endsAt < b.campaign.endsAt) return 1
-
-      if (a.campaign.staked.nativeCurrencyAmount.greaterThan(b.campaign.staked.nativeCurrencyAmount)) return -1
-      if (a.campaign.staked.nativeCurrencyAmount.lessThan(b.campaign.staked.nativeCurrencyAmount)) return 1
-
-      return 0
-    })
-
     return {
       loading: false,
-      miningCampaigns: { active: sortedActiveCampaigns, expired: sortedExpiredCampaigns },
+      miningCampaigns: {
+        active: sortActiveCampaigns(activeCampaigns),
+        expired: sortExpiredCampaigns(expiredCampaigns),
+      },
```

Comment:
Love stuff like this! Really cleans up the code!

Reply:
Thanks Mi-Lan! ✨